core/src/main/java/com/radixdlt/genesis/olympia/converter/OlympiaValidatorsConverter.java

```diff
@@ -99,7 +99,7 @@ private static GenesisValidator convertValidator(
     }
     final var metadata =
         ImmutableList.of(
-            tuple("name", olympiaValidator.name()), tuple("url", olympiaValidator.url()));
+            tuple("name", olympiaValidator.name()), tuple("info_url", olympiaValidator.url()));
```

Comment:
This still isn't right - InfoUrl needs to be of metadata type `Url`, not `String` - we probably need a Metadata type in Java for this?

Reply:
Done in this PR :)